packages/amplify_analytics_pinpoint/android/build.gradle

```diff
@@ -47,8 +47,8 @@ android {
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
 
-    implementation 'com.amplifyframework:aws-analytics-pinpoint:1.0.0'
-    implementation 'com.amplifyframework:aws-auth-cognito:1.0.0'
+    implementation 'com.amplifyframework:aws-analytics-pinpoint:1.4.1'
```

Comment:
Anything special about this version?

https://mvnrepository.com/artifact/com.amplifyframework/core